lib/postrank-uri.rb

```diff
@@ -94,7 +94,7 @@ module URI
         }iox;
 
     URIREGEX[:escape]   = /([^ a-zA-Z0-9_.-]+)/x
-    URIREGEX[:unescape] = /((?:%[0-9a-fA-F]{2})+)/x
+    URIREGEX[:unescape] = /%[0-9a-fA-F]{2}/x
```

Comment:
I think I want to avoid any ambiguity in this case. It's probably exactly equivalent but I'd rather be very explicit here since all this encoding stuff is so crazy.